Terra/Workspaces/BloodPressure_GWAS_Template.md

```diff
@@ -234,3 +224,5 @@ Contributing authors include:
 | December 3, 2019 | Gen3 updates | Beth |
 | November 22, 2019 | Updates from Alisa | Beth |
 | October 22, 2019 | User experience edits from Beri | Beth|
+
+† This was performed in the interest of clarity and cost-savings by minimizing the time spent using Hail.
```

Comment:
Can you just add this statement to the change log above?

Reply:
Yes, but it makes the table take up at lot more space.
<img width="814" alt="Screenshot 2021-01-26 at 12 45 18 PM" src="https://user-images.githubusercontent.com/27784612/105903162-6826a280-5fd4-11eb-9ac4-6bc36cf66f5c.png">